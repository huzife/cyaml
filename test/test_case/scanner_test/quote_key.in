---
"first_key" : 
  "\nsecond_\tkey":
  "third_key" : hello
'\""\': test
...
