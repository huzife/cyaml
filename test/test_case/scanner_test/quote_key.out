(STREAM_START)#
(DOC_START)#
(BLOCK_MAP_START)#
(KEY, first_key)#
(BLOCK_MAP_START)#
(KEY, 
second_	key)#
(SCALAR, )#
(KEY, third_key)#
(SCALAR, hello)#
(BLOCK_MAP_END)#
(KEY, \""\)#
(SCALAR, test)#
(BLOCK_MAP_END)#
(DOC_END)#
(STREAM_END)#
